source/index.html

```diff
@@ -2,11 +2,160 @@
 <html lang="ru">
   <head>
     <meta charset="utf-8">
+    <link rel="icon" type="image/png" sizes="32x32" href="../source/img/icon/favicon.png">
+    <link rel="stylesheet" href="../source/css/">
     <title>HTML Academy: Кэт энерджи</title>
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <h1 class="visually hidden">CAT ENERGY</h1>
```

Comment:
Наверное всё-таки
```suggestion
    <h1 class="visually-hidden">CAT ENERGY</h1>
```
Иначе это будет два разных класса

PS Ещё можно так 😈
```suggestion
    <h1 class="visuallyㅤhidden">CAT ENERGY</h1>
```
но на сдачу лучше так не делать, и вообще не делать